cranelift-codegen/src/binemit/stackmap.rs

```diff
@@ -52,25 +52,23 @@ impl Stackmap {
             }
         }
 
-        Stackmap::from_vec(&vec)
+        Stackmap::from_slice(&vec)
     }
 
     /// Create a vec of Bitsets from a vec of bools.
```

Comment:
nit: this comment now needs an update; it's not a vec anymore.

Reply:
🤦‍♂️ Indeed